internal/learn/iota-token/staking-iota.md

```diff
@@ -0,0 +1,100 @@
+
+### IOTA staking for ASMB rewards
+
+It is planned that IOTA token holders will receive 20% (20 billion tokens) of the total initial Assembly supply of 100 billion tokens. Read more about the Assembly token launch and distribution [in this blog](https://blog.assembly.sc/assembly-fair-launch-token-distribution/).
+- To distribute the first batch of those Assembly tokens to IOTA holders, the staking functionality in the Firefly wallet has been developed. 
+- In the initial 90 days staking period, a maximum of 8% of the Assembly supply can be distributed to IOTA token stakers. 
+- This happens at a fixed rate for every IOTA token staked.
+- This rate has been set to `0,000004 ASMB tokens per 1 MIOTA per milestone`. That means for every 1 MIOTA that you hold and stake in Firefly, you will receive 0,000004 ASMB tokens every 10 seconds (the interval of milestones issued in the IOTA tangle)
+
+![assembly staking](/img/learn/iota-token/assembly-staking.png)
+
+ You can read how to stake and calculate the possible rewards [here](https://assembly.sc/stake).
+- After the 90 days have passed, the total amount of IOTA tokens that have participated in the staking event for Assembly will have created a set amount of Assembly tokens (maximum 8 billion if 100% if the IOTA supply has been staked for Assembly)
+- With the launch of the Assembly network, these generated Assembly tokens will be distributed to the IOTA stakers. This distribution will be implemented as a functionality within the Firefly wallet. As long as the Assembly network is not yet launched, Firefly will hold the record of rewards for the IOTA token holder. 
+
+### IOTA staking for SMR rewards
+
+Shimmer, the incentivized testnet and battlefield for IOTA's core technology innovations, will be bootstrapped purely from IOTA token stakers. The initial token supply of the Shimmer network will only rely on the amount of IOTA tokens staked for Shimmer in the 90-day staking period, and 100% of the initial supply will be distributed to IOTA token holders.
+
+- The same staking functionality used for Assembly in the Firefly Wallet will also be used for Shimmer.
+- Shimmer tokens will be distributed at a fixed rate of `1 Shimmer per 1 MIOTA per milestone`.
+
+![shimmer staking](/img/learn/iota-token/shimmer-staking.png)
+
+You can read how to stake and calculate the possible rewards [here](https://shimmer.network/claim).
+- With the launch of the Shimmer network, these generated Shimmer tokens will be distributed to the IOTA stakers. This distribution will be implemented as a functionality within the Firefly wallet. As long as the Shimmer network is not yet launched, Firefly will hold the record of rewards for the IOTA token holder. 
+
+## Staking IOTA tokens to further bootstrap the Assembly network
```

Comment:
I think we should shorten this header, it looks weird in the table of contents.

Reply:
I still think this should mention "post-launch" as it is a continuation of the "pre-launch" staking period.

Reply:
Now? 😅 Bootstrap isn‘t the right word then imo

Reply:
Its not really supporting either. It's just to finish the distribution of the remainder of the 20% after the pre-launch period. This is all described below, so I'd just name it "Assemy post-launch staking" to remain in the pre-launch header scheme above. Wdyt?

Reply:
Done